Marlin/src/gcode/bedlevel/G26.cpp

```diff
@@ -823,11 +830,17 @@ SERIAL_ECHOLNPAIR("  Z2 at:", current_position[Z_AXIS]);
         //}
 
         print_line_from_here_to_there(rx, ry, g26_layer_height, xe, ye, g26_layer_height);
+        MYSERIAL0.flush(); // G26 takes a long time to complete.   PronterFace can
+                           // over run the serial character buffer with M105's without
+                           // this fix
 
       }
       if (look_for_lines_to_connect())
         goto LEAVE;
     }
+    MYSERIAL0.flush(); // G26 takes a long time to complete.   PronterFace can
```

Comment:
Same comment as above

Reply:
- Have these been addressed?
- Has this been back-ported to 1.1.x? My faith is low.

Reply:
There's lots of places in 2.0 that have gone back to using MYSERIAL0 directly since I cleaned all of them up as part of multi-host. It'd be nice if there were a way to prevent use of MYSERIAL0 and MYSERIAL1 and force use of the macros in serial.h. However, I don't think that's possible since they are macros and not functions.

Reply:
We ought to tweak Marlin 1.1.x and rename `MYSERIAL` to `MYSERIAL0` for easier comparison with 2.0.x.